### PR TITLE
WRN-3473: FixedPopupPanels, PopupTabLayout: Prevent left key navigation when popup opened inside

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
FixedPopupPanels: onBack is called during key navigation in additional popup

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added a guard for the case

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
I'm not sure we need to handle cancel key on sandstone to prevent onBack.
In this PR, I just add handler on sampler side.

### Links
[//]: # (Related issues, references)
WRN-3473

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Fixed
 
+- `sandstone/FixedPopupPanels` and `sandstone/PopupTabLayout` to not move to the previous panel by left key on popup
```
Comment (MikyungKim) [inline CHANGELOG.md:15]:
```suggestion
- `sandstone/FixedPopupPanels` and `sandstone/PopupTabLayout` to not go back to the previous panel by left key on popup opened inside
```

File: samples/sampler/stories/qa/FixedPopupPanels.js [hunk 2]
```diff
@@ -19,9 +21,11 @@ import Pause from '@enact/spotlight/Pause';
 import {Column, Cell} from '@enact/ui/Layout';
 import ri from '@enact/ui/resolution';
 import PropTypes from 'prop-types';
-import {Component, useState} from 'react';
+import {Component, useCallback, useState} from 'react';
 import compose from 'ramda/src/compose';
 
+add('cancel', 27);
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/FixedPopupPanels.js:27]: Let's move this after the config block.

File: samples/sampler/stories/qa/FixedPopupPanels.js [hunk 3]
```diff
@@ -291,6 +302,14 @@ const WithVariousItemsSamplesBase = ({rtl}) => {
 		}
 	};
 
+	const handleKeyUpOnPopup = useCallback((ev) => {
+		if ((isCancel(ev.keyCode)) && popupOpen) {
+			setPopupOpenState(false);
+			ev.stopPropagation();
+			ev.preventDefault();
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/FixedPopupPanels.js:309]:
```suggestion
			ev.preventDefault();
			ev.stopPropagation();
```

File: samples/sampler/stories/qa/PopupTabLayout.js [hunk 4]
```diff
@@ -177,6 +187,14 @@ const WithVariousItemsSamplesBase = ({rtl}) => {
 		}
 	};
 
+	const handleKeyUpOnPopup = useCallback((ev) => {
+		if ((isCancel(ev.keyCode)) && popupOpen) {
+			setPopupOpenState(false);
+			ev.stopPropagation();
+			ev.preventDefault();
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/PopupTabLayout.js:194]:
```suggestion
			ev.preventDefault();
			ev.stopPropagation();
```

**Review by MikyungKim — APPROVED** [review]
LGTM!

**Review by seunghoh — APPROVED** [review]
LGTM for now. 
Let's discuss how to handle `cancel` key later.

**Pull request merged**